docs/sdk-and-tools/indices/scresults.md

```diff
@@ -0,0 +1,56 @@
+---
+id: es-index-scresults
+title: Elasticsearch index: scresults
+---
+
+The `_id` field for this index is composed of hex encoded smart contract result hash.
+(example: `cbd4692a092226d68fde24840586bdf36b30e02dc4bf2a73516730867545d53c`)
+
+
+| Field             | Description                                                                                                                                                    |
+|-------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| miniBlockHash     | The miniBlockHash represents the hash of the miniblock in which the smart contract result was included.                                                        |
+| nonce             | The nonce field represents the transaction sequence number.                                                                                                    |
+| gasLimit          | The gasLimit field represents the maximum gas units the sender is willing to pay for.                                                                          |
+| gasPrice          | The gasPrice field represents the amount to be paid for each gas unit.                                                                                         |
+| value             | The value field represents the amount of EGLD to be sent from the sender to the receiver.                                                                      |
+| sender            | The sender field represents the address of the smart contract result sender.                                                                                   |
+| receiver          | The receiver field represents the destination address of the smart contract result.                                                                            |
+| senderShard       | The senderShard field represents the shard ID of the sender address.                                                                                           |
+| receiverShard     | The receiverShard field represents the shard ID of the receiver address.                                                                                       |
+| relayerAddr       | The relayerAddr field represents the address of the relayer.                                                                                                   |
+| relayedValue      | The relayedValue field represents the amount of EGLD to be sent from the relayed.                                                                              |
```

Comment:
I think this is wrong. This field represents the amount of EGLD to be transferred via the inner transaction's sender, isn't it?

Reply:
yes, you are right. changed.